pkg/codegen/schema/schema.go

```diff
@@ -538,8 +538,12 @@ type Function struct {
 	Comment string
 	// Inputs is the bag of input values for the function, if any.
 	Inputs *ObjectType
+	// Determines whether the input bag should be treated as a single argument or as multiple arguments.
+	MultiArgumentInputs bool
 	// Outputs is the bag of output values for the function, if any.
 	Outputs *ObjectType
+	// The return type of the function, if any.
+	ReturnType Type
```

Comment:
I'm surprised this is just being add as a `Type`, how would we mark the single return value as secret?

Reply:
@Frassle in this iteration, this is not yet possible.

Reply:
Is it going to be harder to add support for that in a later iteration while also supporting Type at this location in the schema? I would have made a return parameter nested object for this, which in this iteration simply has one field type. But if y'all think something like that will be easy to add later with this as it is then fine.

Reply:
@Frassle it can be added as a field to the `Function` definition, not necessarily the return type itself.

Reply:
Well yes I suppose it could but I'd find that distasteful. But being a question of taste and given I don't work much with this stuff I'll just leave it at that.

Reply:
@Frassle it's the same for `PropertySpec`, the secret-ness is not a field of the `Type` of the property but adjacent to it (as a field to the `PropertySpec` itself). Adding `sercret: bool` to `FunctionSpec` would follow the same pattern